apps/fluent-tester/src/RNTester/TestComponents/index.ts

```diff
@@ -1,88 +1,7 @@
 import * as React from 'react';
-import { ButtonFocusTest, HOMEPAGE_BUTTON_BUTTON } from './Button';
-import { CalloutTest, HOMEPAGE_CALLOUT_BUTTON } from './Callout';
-import { CheckboxTest, HOMEPAGE_CHECKBOX_BUTTON } from './Checkbox';
-import { FocusTrapTest, HOMEPAGE_FOCUSTRAPZONE_BUTTON } from './FocusTrapZone';
-import { LinkTest, HOMEPAGE_LINK_BUTTON } from './Link';
-import { PersonaTest, HOMEPAGE_PERSONA_BUTTON } from './Persona';
-import { PersonaCoinTest, HOMEPAGE_PERSONACOIN_BUTTON } from './PersonaCoin';
-import { PressableTest, HOMEPAGE_PRESSABLE_BUTTON } from './Pressable';
-import { RadioGroupTest, HOMEPAGE_RADIOGROUP_BUTTON } from './RadioGroup';
-import { SeparatorTest, HOMEPAGE_SEPARATOR_BUTTON } from './Separator';
-import { SvgTest, HOMEPAGE_SVG_BUTTON } from './Svg';
-import { TextTest, HOMEPAGE_TEXT_BUTTON } from './Text';
-import { ThemeTest, HOMEPAGE_THEME_BUTTON } from './Theme';
 
 export type TestDescription = {
```

Comment:
I would consider figuring out how to add a platform mask as a pattern to the various entries, either as an include list or exclude list. Something like:

```ts
{
  name: 'Button Test',
  component: ButtonFocusTest,
  testPage: HOMEPAGE_BUTTON_BUTTON,
  platforms: ['android', 'ios', 'macos', 'web', 'win32', 'windows']
}
```

Then instead of passing in the set of test descriptions you can pass in a platform setting to the RNTester where it can do filtering. 

The approach you use here is correct and safe, but is a bit tougher to maintain. In particular the authoring of the tests and the references to the tests span multiple packages, and when a test is added or updated it needs to be updated in multiple places.

Reply:
One of the problems we've run into is related to 'bad' imports calling into things that don't exist in react-native-web. When a component imports (directly or indirectly) something like `Callout` or `FocusTrapZone` that calls `requireNativeComponent` we hit runtime type errors as this isn't something that is supplied by react-native-web (the bundler aliases rn with rn-web). I can imagine a couple solutions to this, but being really granular about the imports and where they come from seems to work. Runtime filtering per platform likely results in the problems we've been seeing. @gulnazsayed do you have any additional thoughts on this? I know you're looking at cleaning up some imports in the testComponents directory to work.

We should look into per platform bundling strategies to make stubbing things easier and runtime filtering becomes with that in place.